src/library_syscall.js

```diff
@@ -1815,17 +1823,27 @@ if (SYSCALL_DEBUG) {
   }
 }
 
+var WASI_SYSCALLS = set(['fd_write']);
+
 for (var x in SyscallsLibrary) {
+  var which = null; // if this is a musl syscall, its number
```

Comment:
Can we give this a better name?  How about syscall_nunber?

Reply:
We currently use `which` as the name for the musl syscall number throughout this file (grep says 114 times). I agree it's worth improving that, how about in a separate PR?